frontend/src/Components/Indicatorbars/PlantListBar.js

```diff
@@ -0,0 +1,63 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { ReactComponent as FertilizerIcon } from '../../assets/icons/fertilizericon.svg'
+import { ReactComponent as WaterIcon } from '../../assets/icons/watericon.svg'
+
+export default function PlantListBar({ waterDays, fertilizerWeeks }) {
```

Comment:
I don't have an actual idea how to do this (I'm having kind of the same problem in my app) but this code is already included in PlantDetailBar.js - I'm wondering if there is a way to combine the two but only show the additional details when needed instead of having two.